src/on.ts

```diff
@@ -0,0 +1,93 @@
+import { Handle, EventObject } from './interfaces';
+import { createHandle, createCompositeHandle } from './lang';
+import Evented from './Evented';
+
+// Only used for Evented and EventEmitter, as EventTarget uses EventListener
+type EventCallback = (event: {}) => void;
+
+interface ExtensionEvent {
+	(target: any, listener: EventCallback): Handle;
+}
+
+interface EventTarget {
+	addEventListener(event: string, listener: EventListener, capture?: boolean): Handle;
+	removeEventListener(event: string, listener: EventListener, capture?: boolean): void;
+}
+
+interface EventEmitter {
+    on(event: string, listener: EventCallback): EventEmitter;
+	removeListener(event: string, listener: EventCallback): EventEmitter;
+}
+
+export default function on(target: EventTarget, type: string, listener: EventListener, capture?: boolean): Handle;
+export default function on(target: EventTarget, type: ExtensionEvent, listener: EventListener, capture?: boolean): Handle;
+export default function on(target: EventTarget, type: [string|ExtensionEvent], listener: EventListener, capture?: boolean): Handle;
```

Comment:
This adds support for attaching a single handler for multiple events in one statement. Dojo 1.x used comma-separated string syntax and parsed it accordingly, adding unnecessary complexity. @kriszyp suggested supporting arrays of events in chat in passing. It's easy enough to remove if we want to nix it.